pennylane/interfaces/jax_jit.py

```diff
@@ -206,11 +206,26 @@ def non_diff_wrapper(args):
 
             return (tuple(res),)
 
-        # Gradient function is a device method.
-        with qml.tape.Unwrap(*tapes):
-            jacs = gradient_fn(tapes, **gradient_kwargs)
+        def wrapper(p):
+            new_tapes = []
+            for t, a in zip(tapes, p):
+                new_tapes.append(t.copy(copy_operations=True))

```

Comment:
[This is what I found on this.](https://github.com/PennyLaneAI/pennylane/pull/1634#discussion_r714702281)

Reply:
Great! :star_struck: